hazelcast/src/test/java/com/hazelcast/jet/impl/connector/WriteJdbcPTest.java

```diff
@@ -176,22 +173,25 @@ public void test() throws SQLException {
     public void test_supplied_closeable_datasource_is_closed() throws SQLException {
         Pipeline p = Pipeline.create();
 
+        CopyOnWriteArrayList<HikariDataSource> hikariDataSourceList = new CopyOnWriteArrayList<>();
+
         p.readFrom(TestSources.items(IntStream.range(0, PERSON_COUNT).boxed().toArray(Integer[]::new)))
                 .map(item -> entry(item, item.toString()))
                 .writeTo(Sinks.jdbc("INSERT INTO " + tableName + " VALUES(?, ?)",
-                        WriteJdbcPTest::createHikariDataSource,
+                        () -> createHikariDataSource(hikariDataSourceList),
```

Comment:
And here will be
`() -> hikariDataSource`

Reply:
The same explanation as above. We need to use a list